wiki/Guides/Using_custom_hitsounds/de.md

```diff
@@ -1,49 +1,68 @@
+4. Klicke auf `Datei` (die Option am linken Ende des Navigations-Menüs).
+5. Klicke auf `Songs-Ordner öffnen`.
+6. Füge deinen Dateien hier ein.
 
-1. Öffne osu!
-2. Klicke auf Edit.
-3. Navigiere zu *deine Beatmap hier* und öffne sie.
-4. Klicke oben auf Datein (Files) (Das Menu oben-links, es ist die weiß gefärbte Leiste oben).
-5. Öffne den Song Ordner.
-6. *und füge die Hitsounds ein*
+Falls du osu! auf macOS ausführst, musst du vielleicht eine leicht andere Vorgehensweise wählen:
 
-## Die Wahl welche Art von Ton der Hitsound sein soll
+1. Rechtsklicke auf das osu!-Anwendungsicon und wähle `Paketinhalt anzeigen`.
+2. Finde den Ordner deiner Beatmap in `drive_c -> osu! -> Songs`.
+3. Füge deine Dateien hier ein.
 
-Die Namensgebung ist für "finish", "whistle", "clap" und "normal hit" egal.
+Wenn die Tondateien im Ordner der Beatmap platziert sind, müssen sie angemessen benannt werden, damit osu! sie als Hitsounds erkennt.
 
-Abhängig von welcher Art von den Hitsound man haben möchte, sollte der Dateiname mit "soft", "normal" oder "drum" beginnen.
+In osu! existieren drei Kategorien von Hitsounds, die als *Samplesets* bekannt sind: Normal (N), Soft (S), und Drum (D). Jedes Sampleset kann weiterhin in verschiedene Töne unterteilt werden. Die häufigsten sind: "hitnormal", "hitclap", "hitwhistle" und "hitfinish". Außerdem existieren speziellere Töne, wie die, die während eines Sliders ("sliderslide", slidertick) oder eines Spinners ("spinnerspin") abgespielt werden.
 
-![Tutorial Image 1](img/beatmap-folder-resources.png "Tutorial Image 1")
+*Für eine vollständige Liste an Hitsounds, die modifiziert werden können, siehe: [Skinning-Eintrag zu Hitsounds](/wiki/Skinning/Sounds#hitsounds)*
 
-### Beispiel 1
+Hitsound-Dateien geben die beiden Eigenschaften des Samplesets und des Tontyps folgendermaßen im Namen wieder:
 
-Man hat die Datei (Sagen wir mal Soft clap) `soft-hitclap.wav` genannt.
+`<sampleset>-<ton>.wav`
 
-## Mehrere Hitsound mit den selben Namen
+Hier ist `<sampleset>` entweder "normal", "soft" oder "drum" und `<sound>` ist eine der oben genannten Unterteilungen (z. B. "hitclap").
 
-Wenn mehr Abwechslung gebraucht oder mehr Optionen für Clap und so weiter benötigt wird, kann man einfach eine Zahl dahinter schreiben.
+![](img/beatmap-folder-resources.png "Ein typischer Beatmap-Ordner, der eigene Hitsounds enthält")
 
-### Beispiel 2
+Im oben gezeigten Bild ist der erste aufgelistete Ton mit `soft-hitclap.wav` benannt. Dieser wird den Standard-Hitsound, der beim Treffen eines [Hit-Objekts](/wiki/Hit_Object) mit dem "Soft"-Sampleset und der "hitclap"-Unterteilung abgespielt wird, ersetzen. Beachte, dass dieser Ton nur im *ausgewählten Sampleset*  abgespielt wird. Falls deine Beatmap andere Samplesets verwendet, erfordern diese zusätzliche Hitsound-Dateien (auch dann, wenn du genau das gleiche Tonsample verwendest), zum Beispiel eine `normal-hitclap.wav`, während du das "Normal"-Sampleset verwendest.
```

Comment:
```suggestion
Im oben gezeigten Bild ist der erste aufgelistete Ton mit `soft-hitclap.wav` benannt. Dieser wird den Standard-Hitsound, der beim Treffen eines [Hit-Objekts](/wiki/Hit_Object) mit dem "Soft"-Sampleset und der "hitclap"-Unterteilung abgespielt wird, ersetzen. Beachte, dass dieser Ton nur im *ausgewählten Sampleset* abgespielt wird. Falls deine Beatmap andere Samplesets verwendet, erfordern diese zusätzliche Hitsound-Dateien (auch dann, wenn du genau das gleiche Tonsample verwendest), zum Beispiel eine `normal-hitclap.wav`, während du das "Normal"-Sampleset verwendest.
```
I like to write phrases like `"hitclap" addition` as `der Unterteilung "hitclap"` because I think it's more readable. Though, I will let you decide on this. It's not that important.